📄✨：colon etiquette, general style guidance (#1137)
OpenINF/openinf.github.io: collections/_docs/handbook/style/colons.md

---
title: Colons
key_point: A colon indicates that closely-related information follows.
original_url: <https://developers.google.com/style/colons>
original_title: Colons | Google developer documentation style guide
---

<section id="phrase-preceding-colon" markdown="1">

## Introductory Phrase Preceding Colon

When a colon introduces a list, the text that precedes the colon must be able to
stand alone as a complete sentence.

<strong>Examples</strong>

<p class="example">
  <span class="compare-better">Recommended:</span> The fields are defined as
  follows:
</p>
<p class="example">
  <span class="compare-worse">Not recommended:</span> The fields are:
</p>

</section>

<section id="usage-of-japanese-colon" markdown="1">

## Proper Etiquette for Use of Japanese Colon

We at OpenINF are always interested in finding ways to optimize things and
be more efficient. One particular use-case of [the Japanese colon][] that we

> have picked- up is immediately after the classification portion of a commit

Author: ```suggestion
have picked-up is immediately after the classification portion of a commit
```

message title.

Rather than using an ordinary colon from ANSI letters, we use the Japanese

> colon.

Author: ```suggestion
colon as it is both less size (in memory) and can accomplish the task of both
a colon character and the subsequent space all at once.
```

---
title: Colons
key_point: A colon indicates that closely-related information follows.
original_url: <https://developers.google.com/style/colons>
original_title: Colons | Google developer documentation style guide
---

<section id="phrase-preceding-colon" markdown="1">

## Introductory Phrase Preceding Colon

> When a colon introduces a list, the text that precedes the colon must be able to

Author: ```suggestion
When a colon introduces a list, the text that precedes the colon should ordinarily be able to
```

ordinarily be able to stand alone as a complete sentence.

<strong>Examples</strong>

<p class="example">
  <span class="compare-better">Recommended:</span> The fields are defined as
  follows:
</p>
<p class="example">
  <span class="compare-worse">Not recommended:</span> The fields are:
</p>

</section>

<section id="usage-of-japanese-colon" markdown="1">

## Proper Etiquette for Use of Japanese Colon

We at OpenINF are always interested in finding ways to optimize things and be

> more efficient. One particular use-case of [the Japanese colon][] that we have

Author: ```suggestion
more efficient. One particular use case of [the Japanese colon][] that we have
```

picked-up is immediately after the classification portion of a commit message
title.

> Rather than using an ordinary colon from ANSI letters, we use the Japanese colon

Author: ```suggestion
Rather than using an ordinary colon from the ANSI or Unicode character sets, we use the Japanese colon
```

as it is both less size (in memory) and can accomplish the task of both a colon
character and the subsequent space all at once.

</section>

<section id="bold-text-preceding-colon" markdown="1">

## Bold and Italic Text Preceding Colons

> When non-italic (_roman_) text that precedes a colon is bold, the colon should

Author: ```suggestion
When non-italic (also known as _Roman_) text that precedes a colon is bold, the colon should
```

> colon should not be made bold, italic, or otherwise (color-stylized, etc.).

Author: ```suggestion
colon should not be made bold, italic, or otherwise (color-stylized, etc.).
```

is this contentious

Author: ```suggestion
colon should not be made bold, italic, or otherwise (color-stylized, etc.).
This, however, is not a hard and fast rule. In general, use best intuition.
```

## Colons Within Sentences

In general, the first word in the text that follows a colon should be in
lowercase. For exceptions, see [capitalization][].

<strong>Examples</strong>

<p class="example">
  <span class="compare-better">Recommended:</span> Tone: concise,
  conversational, friendly, respectful
</p>
<p class="example">
  <span class="compare-better">Recommended:</span> When you add or update
  content to an existing project, remember to take these steps: review the style
  guide, use checklists, enlist a fellow writer or an editor to copyedit your
  work, and request a developmental edit if you feel it's warranted.
</p>

</section>

## See Also

For more information about how to punctuate introductory material, see the
sections on [list introductions][] and [code-sample introductions][].

For information about when it's better to use colons than dashes, see
[Dashes][].

<!-- LINK DEFINITION LABELS - START -->

[Some specific items to put in code font]:
  ./code-in-text.md#some-specific-items-to-put-in-code-font
[capitalization]: ./capitalization.md
[list introductions]: ./lists.md#intros
[code-sample introductions]: ./code-samples.md#intros
[Dashes]: ./dashes.md#colons

> [the Japanese colon]: https://en.wikipedia.org/wiki/Japanese_punctuation#Colon

Author: ```suggestion
[the Japanese colon]: https://wikipedia.org/wiki/Japanese_punctuation#Colon
```

---
title: Colons
key_point: A colon indicates that closely-related information follows.
original_url: <https://developers.google.com/style/colons>
original_title: Colons | Google developer documentation style guide
---

> <section id="phrase-preceding-colon" markdown="1">

Author: ```suggestion
<section id="phrase-preceding-colon">
```

Author: i have been trying to remember why these are necessary

## Introductory Phrase Preceding Colon

> When a colon introduces a list, the text that precedes the colon should
> ordinarily be able to stand alone as a complete sentence.

Author: ```suggestion
When a colon introduces a list, the text that precedes the colon _**should**
ordinarily_ be able to stand alone as a complete sentence.
```

<strong>Examples</strong>

<p class="example">
  <span class="compare-better">Recommended:</span> The fields are defined as
  follows:
</p>
<p class="example">
  <span class="compare-worse">Not recommended:</span> The fields are:
</p>

</section>

<section id="usage-of-japanese-colon" markdown="1">

## Proper Etiquette for Use of Japanese Colon

We at OpenINF are always interested in finding ways to optimize things and be

> more efficient. One particular use case of [the Japanese colon][] that we have

Author: ```suggestion
more efficient. One particular use case of **[the Japanese colon][]** that we have
```

> picked-up is immediately after the classification portion of a commit message

Author: ```suggestion
picked-up is in its placement immediately after the _**[Classification][]** portion_ in the titles of commit messages written sure to be abiding by our commit message format and style guidelines*hellip;
```

title.

> Rather than using an ordinary colon from the ANSI or Unicode character sets, we

Author: ```suggestion
Rather than using an ordinary colon from the ANSI, ASCII, or Unicode character sets, we
```

use the Japanese colon as it is both less size (in memory) and can accomplish
the task of both a colon character and the subsequent space all at once.

</section>

<section id="bold-text-preceding-colon" markdown="1">

## Bold and Italic Text Preceding Colons

> When non-italic (also known as _Roman_) text that precedes a colon is bold, the

Author: ```suggestion
When _non-italic_ (also known as _Roman_) text (that precedes a colon is **bold**), the
```

> colon should not be made bold, italic, or otherwise (color-stylized, etc.).

Author: ```suggestion
colon _should **not**_ be made_ bold_, _italic_, or _otherwise_ (e.g., _color-stylized_, etc.).
```

> This, however, is not a hard and fast rule. In general, use best intuition.

Author: ```suggestion
This, however, _is **not**_ a hard and fast rule. In general, use _best intuition_.
```

</section>

<section id="code-text-preceding-colon" markdown="1">

## Code Text Preceding Colons

When text that precedes a colon is tagged as `<code>`, don't include the colon

> in the `<code>` tagging unless it is part of the code. For more information

Author: ```suggestion
in the **`<code>`** tagging_ ***unless*** it is _already_ part of the code itself. For more information
```

> about formatting code, see [Some specific items to put in code font][].

Author: ```suggestion
about formatting code, see **[Some specific items to put in code font][]**.
```

</section>

<section id="colon-in-sentence" markdown="1">

## Colons Within Sentences

In general, the first word in the text that follows a colon should be in

> lowercase. For exceptions, see [capitalization][].

Author: ```suggestion
lowercase. For exceptions, see **[capitalization][]**.
```

---
title: Colons
key_point: A colon indicates that closely-related information follows.
original_url: <https://developers.google.com/style/colons>
original_title: Colons | Google developer documentation style guide
---

<section id="phrase-preceding-colon">

## Introductory Phrase Preceding Colon

When a colon introduces a list, the text that precedes the colon _**should**
ordinarily_ be able to stand alone as a complete sentence.

<strong>Examples</strong>

<p class="example">
  <span class="compare-better">Recommended:</span> The fields are defined as
  follows:
</p>
<p class="example">
  <span class="compare-worse">Not recommended:</span> The fields are:
</p>

</section>

<section id="usage-of-japanese-colon" markdown="1">

## Proper Etiquette for Use of Japanese Colon

We at OpenINF are always interested in finding ways to optimize things and be
more efficient. One particular use case of **[the Japanese colon][]** that we
have picked-up is in its placement immediately after the _**[Classification][]**
portion_ in the titles of commit messages written sure to be abiding by our

> commit message format and style guidelines\*hellip; title.

Author: ```suggestion
commit message format and style guidelines.
```

Rather than using an ordinary colon from the ANSI, ASCII, or Unicode character
sets, we use the Japanese colon as it is both less size (in memory) and can
accomplish the task of both a colon character and the subsequent space all at
once.

</section>

<section id="bold-text-preceding-colon" markdown="1">

## Bold and Italic Text Preceding Colons

When _non-italic_ (also known as _Roman_) text (that precedes a colon is
**bold**), the colon _should **not**_ be made* bold*, _italic_, or _otherwise_
(e.g., _color-stylized_, etc.). This, however, _is **not**_ a hard and fast
rule. In general, use _best intuition_.

</section>

<section id="code-text-preceding-colon" markdown="1">

## Code Text Preceding Colons

When text that precedes a colon is tagged as `<code>`, don't include the colon

> in the **`<code>`** tagging\* **_unless_** it is _already_ part of the code

Author: ```suggestion
in the **`<code>`** tagging **_unless_** it is _already_ part of the code
```

> in the **`<code>`** tagging\* **_unless_** it is \_already\* part of the code

Author: ```suggestion
in the **`<code>`** tagging\* **_unless_** it is _already_ part of the code
```
